backend/fms_core/template_importer/importers/sample_update.py

```diff
@@ -0,0 +1,57 @@
+from fms_core.models import Process, Protocol
+
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.sample_update import SampleRowHandler
+
+class SampleUpdateImporter(GenericImporter):
+    SHEETS_INFO = [
+        {'name': 'SampleUpdate', 'header_row_nb': 5},
+    ]
+
+    def __init__(self):
+        super().__init__()
+        # Preload objects accessible to the whole template (not only by row)
+        self.preload_data_from_template()
+
+    def preload_data_from_template(self):
+        self.preloaded_data = {'process': None}
+
+        self.preloaded_data['process'] = Process.objects.create(protocol=Protocol.objects.get(name="Update"),
+                                                                comment="Updated samples (imported from template)")
+
+    def import_template_inner(self):
+        print('Import Sample Update Sheet - import template inner')
+        sampleupdate_sheet = self.sheets['SampleUpdate']
+
+        for row_id, row_data in enumerate(sampleupdate_sheet.rows):
+            sample = {
+                'coordinates': row_data['Coord (if plate)'],
+                'container': {'barcode': row_data['Container Barcode']}
+            }
+            sample_updated = {
+                'volume': row_data['New Volume (uL)'],
+                'concentration': row_data['New Conc. (ng/uL)'],
+                'depleted': row_data['Depleted'],
+            }
+
+            process_measurement = {
+                'process': self.preloaded_data['process'],
+                'volume_used': row_data['Delta Volume (uL)'],
```

Comment:
Delta Volume is an alternative to the new Volume update. The volume used is either the  `- delta volume` or `old_volume - new_volume`.